employee-rostering-gwtui/src/main/resources/org/optaweb/employeerostering/gwtui/client/resources/i18n/OptaWebUIConstants.properties

```diff
@@ -83,6 +83,11 @@ Rotation.dateHeader=Day {0}
 RotationTimeSelector.rotationDayNumber=Rotation Day #
 RotationTimeSelector.time=Time
 Save=Save
+ServerSideException.entityNotFound=The object you requested was not found: {0}
+ServerSideException.illegalArgument=The parameters you passed in were not valid: {0}
+ServerSideException.nullPointer=It appears the developer made a mistake somewhere. Please contact them.
```

Comment:
Again minor issue - single developer made a mistake, but you ask user to contact "them". Should be either "developers" or "him".

Reply:
Will change to "developers", but "them"/"they" can be used as a singular pronoun: https://en.wikipedia.org/wiki/Singular_they

Reply:
@Christopher-Chianelli your call - you're a native speaker